ValveResourceFormat/Serialization/VfxEval.cs

```diff
@@ -438,13 +473,18 @@ private static string Trimb(string exp)
         }
 
         // naming external variables UNKNOWN, UNKNOWN2, UNKNOWN3,.. where not found
+        // if showMurmurForUnknowns is enabled return them in the form UNKNOWN[e46d252d]
         private string GetExternalVarName(uint varId)
         {
             ExternalVarsReference.TryGetValue(varId, out var varKnownName);
             if (varKnownName != null)
             {
                 return varKnownName;
             }
+            if (showMurmurForUnknowns)
```

Comment:
Shouldn't this go inside the if to know that its actually unknown?

And is there any reason not to always return murmurs?

Reply:
Yea, always returning the murmur is good idea, will make this change